onadata/libs/utils/logger_tools.py

```diff
@@ -241,6 +241,11 @@ def save_attachments(xform, instance, media_files):
                 mimetype=content_type,
                 name=filename,
                 extension=extension)
+    if remove_deleted_media:
+        Attachment.soft_delete(
```

Comment:
Why are we passing the queryset still ? Seems like the `soft_delete` function signature changed to `(instance, user: Optional)` ?